### ISSUE TITLE
Install error on Mac: "rack requires Ruby version >= 2.2.2"

### ISSUE THREAD

**Issue by rmtheis** [skywinder/github-changelog-generator#425]
I'm getting this error when I do `sudo gem install github_changelog_generator` on Mac OS X El Capitan Version 10.11.6:

    ERROR:  Error installing github_changelog_generator:
	rack requires Ruby version >= 2.2.2.

Any suggestions?

**Comment by olleolleolle** [comment]
I think the README wants to help you - you can add a Gemfile in the folder where you want to run the tool. 

If you can work with me to improve those instructions, we can get them to a place where they're for everyone.

Try following the FAQ item about Gemfile - then tell me about your experience. It's never been run by someone trying it for the first time. 

Best of luck!

**Comment by olleolleolle** [comment]
Bundler, the package management gem, which allows you to peg dependencies to versions, has an open issue which is "help OS X users to know what to do".

https://github.com/bundler/bundler/issues/4986

**Comment by olleolleolle** [comment]
In order for you to get ahead, 

This blog post does two things at the beginnging: https://gorails.com/setup/osx/10.11-el-capitan

- Installing Homebrew
- Installing Ruby

Can you do those? Then, you should be able to 

```shell
gem install github_changelog_generator
```

note the absence of the `sudo`.

**Comment by rmtheis** [comment]
Thanks for pointing me in the right direction! In the end I updated Ruby rather than creating a gemfile.

At first I hadn't fully understood the problem, and I hadn’t realized that I’d need to have RVM in order to update my Ruby version. Once I realized that, I was able to install RVM with

    gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
    \curl -L https://get.rvm.io | bash -s stable

and updating my .bash_profile. Then I was able to update Ruby using 

    rvm install ruby-2.3.1

after which the github-changelog-generator install was successful.
